README.md

```diff
@@ -184,3 +184,31 @@ browser at [http://wifi.config](http://wifi.config/) or
 [http://192.168.0.1/](http://192.168.0.1/). If you've configured an SSL
 certificate, it's possible to use `https`. You may also need to change the
 `:dns_name` configuration to match the name on your SSL certificate.
+
+## Callbacks when Starting/Stopping
+
+There may be cases that your application needs to know when the wizard
+is started and/or stopped (i.e. need to stop an app using the same port
+before the wizard starts, etc).
+
+For that, you can use the `:on_start` and `:on_exit` options when calling
+`VintageNetWizard.run_wizard/1` and specify a `{module, function, args}` tuple of the function to call during these events.
+
+```elixir
+defmodule MyApp do
+  def start_wizard() do
+    VintageNetWizard.run_wizard(
+      on_start: {__MODULE__, :handle_on_start, []},
```

Comment:
I don't think `on_start` is needed since presumably if you're calling `VintageNetWizard.run_wizard/1`, you could call `handle_on_start` yourself.